cert-manager.yaml.template

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: nfd-ca-key-pair
+  namespace: node-feature-discovery
+data:
+  tls.crt: <"cat tls.crt|base64 -w 0" encoded output from, e.g, openssl req -x509 -new -nodes -key tls.key -subj "/CN=nfd-ca" -days 10000 -out tls.crt>
+  tls.key: <"cat tls.key|base64 -w 0" encoded output from, e.g, openssl genrsa -out tls.key 2048>
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: nfd-ca-issuer
+  namespace: node-feature-discovery
+spec:
+  ca:
+    secretName: nfd-ca-key-pair
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: nfd-master-cert
+  namespace: node-feature-discovery
+spec:
+  secretName: nfd-master-cert
+  subject:
+    organizations:
+    - node-feature-discovery
+  commonName: nfd-master
+  dnsNames:
+  - nfd-master.node-feature-discovery.svc
+  - nfd-master.node-feature-discovery.svc.cluster.local
+  - nfd-master
+  issuerRef:
+    name: nfd-ca-issuer
+    kind: Issuer
+    group: cert-manager.io
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: nfd-workers-cert
+  namespace: node-feature-discovery
+spec:
+  secretName: nfd-worker-cert
+  subject:
+    organizations:
+    - node-feature-discovery
+  commonName: nfd-worker
+  dnsNames:
+  - nfd-worker.node-feature-discovery.svc.cluster.local
```

Comment:
I don't think this is needed

Reply:
the doc says "# At least one of a DNS Name, URI, or IP address is required." so I added a default value. ideally, we'd get node name here automatically but I could not yet find a way to do it.